packages/storage-plugin/tests/storage.plugin.spec.ts

```diff
@@ -360,6 +361,7 @@ describe('NgxsStoragePlugin', () => {
     store
       .select((state: any) => state)
       .subscribe((state: { counter: StateModel; lazyLoaded: StateModel }) => {
+        console.error('should merge state', state);
```

Comment:
why error?

Reply:
that shouldn't be in there, ill remove that tomorrow morning